### ISSUE TITLE
Jaws of the Lion: Scenario 10 Missing Rewards

### ISSUE THREAD

**Issue by sahilthapar** [Lurkars/gloomhavensecretariat#584]
### Describe the bug

The Scenario 10 rewards 15 xp to each player as a reward which is missing from the data.

### Game Enviroment

Game: Gloomhaven: Jaws of the Lion
Scenario: 10

### To Reproduce

Compare against the scenario book conclusion

### Expected behavior

Rewards should award 15 xp to each player

### Gloomhaven Secretariat Version

0.99.7

### Used Browser

Firefox 127.0 (64-bit)

### OS

macos 13.2

### Additional

I'll submit a PR for this shortly
